### DISCUSSION TITLE
FirebaseAnalytics & Required Reason APIs

### DISCUSSION THREAD

**Discussion by macdrevx** [firebase/firebase-ios-sdk#12501] [General]
I don't see a privacy manifest in FirebaseAnalytics as of 10.22.0. While it's not on Apple's [list](https://developer.apple.com/support/third-party-SDK-requirements/), since it's closed source, I can't tell whether it uses any [required-reason APIs](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api?language=objc). Could someone from the team confirm whether it uses any required-reason APIs?

**Comment by macdrevx** [comment]
ditto for GoogleAppMeasurement & GoogleAppMeasurementIdentitySupport

**Comment by paulb777** [comment]
With version 10.24.0, FirebaseAnalytics, GoogleAppMeasurement, and GoogleAppMeasurementIdentitySupport do not use required reason APIs.
